src/server.js

```diff
@@ -18,6 +18,11 @@ var authFunction = localAuth;
 module.exports = function() {
 	manager = new ClientManager();
 
+	if (!fs.existsSync("client/js/bundle.js")) {
```

Comment:
Solutions I can think of are not ideal. For example, I don't know if this is possible without forcing people committing on the client to include such marker in their PR.

Either way, can we do this in a latter PR please? This PR should not go further than what it currently does.